src/tasks/tap/link.js

```diff
@@ -66,17 +66,28 @@ const buildTapObj = async (globalConfig, silent, name, location) => {
  * @returns {void}
  */
 const linkTap = async args => {
-  const { command, globalConfig, options, params, tasks } = args
+  const { globalConfig, params } = args
+
   const { name, location, silent } = params
 
+  // get the tap config located at our current location to determine the default alias name
+  const [ tapConfig ] = getTapConfig({ path: location })
+
+  // get the tap name, or throw an error if not specified
+  const alias = name || get(tapConfig, 'keg.alias')
+  if (!alias) {
```

Comment:
I'm not sure I explained myself properly for `ask-it` and ci
I was trying to say, I want to update the `ask-it` repo to check the current env. If it's in `ci` or something like that, then don't ask for user input.

So like you mentioned, it would only be done in one place... the `ask-it` repo.
Anything that consumes it, like the `keg-cli` doesn't know or care. It just handles the response
Which in a CI environment, could be `undefined`

Reply:
Ah yeah that could be done, but `ask-it` is pretty decoupled from the `keg-cli`; it shouldn't really care what environment it is in, no? 

We could do it, I just think logic is more suited for the `keg-cli`. 

Really, all we need is to **throw** an error  if `ask-it` gets called in a `ci` env, and we could implement that as a wrapper of `ask-it` like above.